src/app/auth/auth.module.ts

```diff
@@ -0,0 +1,29 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { RouterModule } from '@angular/router';
+
+import { SharedModule } from '@app/shared/shared.module';
+
+import {
+  LoginComponent,
+  LogoutComponent
+} from '@app/auth/pages';
+
+import { authRoutes } from './auth.routes';
+
+@NgModule({
```

Comment:
Why don't you provide `AuthService` here?

Reply:
I read DOCs from https://angular.io/guide/providers
and I use
@Injectable({
  providedIn: 'root'
}) for AuthService, and after 
"You can now inject UserService anywhere in your application."